host/traceh.c

```diff
@@ -24,6 +25,8 @@
 #if defined(__linux__)
 #define sprintf_s(buffer, size, format, argument) \
     sprintf(buffer, format, argument)
+#else
+#include <openenclave/internal/time.h>
```

Comment:
Why is this inside an else clause?  Would be more readable to unconditionally include it.

Reply:
Fixed, thanks!